semantic/src/site/collections/table.overrides

```diff
@@ -2,89 +2,165 @@
-  }
+    .ui.table td:last-child,
+    .ui.table thead th:last-child {
+      text-align: right;
+      padding-right: @tableRowLeftAndRightPadding;
+    }
 
-  .ui.sortable.table thead th {
-    border: none;
-    padding-right: 3px;
-  }
+    // Prevents column headers having different background when selected
+    // and on hover
+    .ui.sortable.table thead th.sorted {
+      background: none;
+    }
+    .ui.sortable.table thead th:hover {
+      background: none;
+      opacity: 0.8;
+    }
 
-  .ui.table tr td {
-    border: none;
-  }
+    // Removes the faint background that is visibile behind the list row
+    // when hovering
+    .ui.selectable.table tbody tr:hover,
+    .ui.table tbody tr td.selectable:hover {
+      background: none !important;
+      color: @selectableTextColor !important;
+    }
+
+    // Column header text
+    .ui.table thead tr th {
+      border: none;
+      background: none;
+      font-weight: @semiboldWeight;
+      font-size: @fontXsmall;
+      letter-spacing: @letterSpacingSmall;
+      text-transform: uppercase;
+      color: @headersInteractiveMed !important;
+    }
+
+    .ui.sortable.table thead th {
+      border: none;
+      padding-right: 3px;
+    }
 
-  .ui.table tr td:first-child {
-    border-top-left-radius: 10px;
-    border-bottom-left-radius: 10px;
+    .ui.table tr td {
+      border: none;
+    }
+
+    .ui.table tr td:first-child {
+      border-top-left-radius: 10px;
+      border-bottom-left-radius: 10px;
+    }
+
+    .ui.table tr td:last-child {
+      border-top-right-radius: 10px;
+      border-bottom-right-radius: 10px;
+    }
   }
+  /*--------------------------------
+      Data views
+  ---------------------------------*/
+
+  #data-view {
+    .flex-column();
+    align-items: center;
+    width: @applicationListWidth;
+
+    .ui.header {
+      padding-top: 30px;
+    }
+
+    .data-view-nav {
+      width: 90%;
+    }
+
+    .data-view-table-container {
+      .flex-column();
+      align-items: center;
+      // Limit size of images in Table (that come from SummaryView elements)
+      .ui.image {
+        width: unset;
+        max-height: 50px;
+      }
+      padding-top: 0;
+      width: clamp(700px, 90%, 1127px);
+    }
+
+    .summary-page-element {
+      margin: 10px;
+      min-width: 200px;
+      max-width: 450px;
+    }
+
+    .data-view-detail-container {
+      .flex-column();
+      align-items: center;
+      // Limit size of images in Table (that come from SummaryView elements)
+      .ui.image {
+        width: unset;
+        max-height: 100px;
+      }
+    }
 
-  .ui.table tr td:last-child {
-    border-top-right-radius: 10px;
-    border-bottom-right-radius: 10px;
+    .data-view-detail-table {
+      min-width: 500px;
+    }
+
+    #linked-applications-container {
+      margin-top: 45px;
+    }
   }
 }
+
+// mobile and small tablet overrides
+
+@media only screen and (max-width: @largestMobileScreen - 1) {
```

Comment:
My linter is complaining about the `-` sign here. Should it be `max-width: calc(@largestMobileScreen - 1)`?

Reply:
That is probably a lot tidier.  Implemented.